content/manuals/build/concepts/dockerfile.md

```diff
@@ -164,23 +149,17 @@ the container.
 
 ### Comments
 
-Note the `# install app dependencies` line. This is a comment. Comments in
-Dockerfiles begin with the `#` symbol. As your Dockerfile evolves, comments can
-be instrumental to document how your Dockerfile works for any future readers
-and editors of the file, including your future self!
+The `# install app dependencies` line is a comment. Comments in Dockerfiles start
+with `#`. Comments help document your Dockerfile for future readers and editors.
```

Comment:
It makes assumption and I preferred the previous one that explains the purpose and meaning of comments. I think we should revert this.